### ISSUE TITLE
Improve Responsive navigation 

### ISSUE THREAD

**Issue by SaraVieira** [styleguidist/react-styleguidist#424]
When on mobile navigation should be a pop up menu on the side but in this case it just gets pushed to the bottom in a somewhat weird way. 

![screenshot_20170502-110514](https://cloud.githubusercontent.com/assets/1051509/25613856/abb5d354-2f27-11e7-98fb-ca8c4394fadd.png)

Open to suggestions here but I think having slide out menu when there is no more room would be the way to go, thoughts? 
.I will work on this if it's not a problem

**Comment by sapegin** [comment]
There are two things:

1. We’ve never spend much time on mobile experience so it’s very simplistic now.

2. There’s more generic UI issue: we’re keep adding things to the UI and right now it’s a bit cluttered and inconsistent.

So any improvements in both are very appreciated but try not to fix just one issue but think a bit broader ;-)

**Comment by SaraVieira** [comment]
I'll try add a pixel grid to this to make the elements more evenly spaced. 
Also put the logo in the header would be awesome :) 

Should I just work with freedom and put the screenshots here?

**Comment by sapegin** [comment]
I was thinking about adding something [like this](https://medium.freecodecamp.com/css-in-javascript-the-future-of-component-based-styling-70b161a79a32) (see *Define spacing constants*).

> Should I just work with freedom and put the screenshots here?

Yeah! 🍕

**Comment by SaraVieira** [comment]
That is a great idea , that's exactly what I was thinking about, using a 8px grid and everything 😄 

Thank you for the push acess 🎉

**Comment by SaraVieira** [comment]
@sapegin The designer in my team sent me this and I think this would be awesome:

![kapture_2017-05-02_at_18 24 48](https://cloud.githubusercontent.com/assets/1051509/25657229/bfb5dcdc-2ff4-11e7-9108-c50eecc0ab13.gif)

**Comment by sapegin** [comment]
But we already have almost the same, but without animation ;-)

I have a proposal for UI improvements but I need some time to write it, will try to open an issue today.

**Comment by SaraVieira** [comment]
@sapegin If you create it today I can just close this ticket and just work in the other one 👍

**Comment by SaraVieira** [comment]
@sapegin It's the theme that makes it look awesome

**Comment by sapegin** [comment]
If you mean dark editor theme, you can change theme for your style guide ;-)

**Comment by SaraVieira** [comment]
I know you can change it but I think it would be a better default

**Comment by sapegin** [comment]
No, sorry, I won’t make a dark theme a default choice — so much harder to read than a light one ;-)

**Comment by SaraVieira** [comment]
true , it looks nicer but it is harder to read

**Comment by sapegin** [comment]
For many developers it’s not but I think for a style guide it’s a better default ;-)
